package.json

```diff
@@ -26,20 +26,26 @@
   },
   "main": "./lib/ffi",
   "dependencies": {
-    "bindings": "^1.3.0",
     "debug": "^3.1.0",
     "get-uv-event-loop-napi-h": "^1.0.5",
     "node-addon-api": "1.6.1",
+    "node-gyp-build": "^4.2.1",
     "ref-napi": "^1.4.0",
     "ref-struct-di": "^1.1.0"
   },
   "devDependencies": {
+    "bindings": "^1.3.0",
```

Comment:
Can be removed?

Reply:
No, it's still used in the tests: https://github.com/node-ffi-napi/node-ffi-napi/blob/master/test/binding.gyp. Since we don't want to prebuild any of the test code I've left it as it was.

Reply:
If a normal build exists, `node-gyp-build` will load that instead of the prebuild.

Reply:
For some reason I thought I'd tested this and `bindings` was necessary for the tests, but I've just tried again that does seem to work with `node-gyp-build` after all :shrug:.

Now removed, we're `node-gyp-build` all the way down.